### PR TITLE
MTM-60386-Missing-info-about-required-permissions-to-use-Smart-Rules-relying-on-SMS-provider

### PR DESCRIPTION
**Pull request by pmic-cumulo**

### CODE REVIEW

File: content/cockpit/smart-rules-collection.md [hunk 1]
```diff
@@ -25,7 +25,9 @@ In certain rule parameters, various trigger fields can be used as variables, see
 If an alarm is created, an SMS is sent.
 
 {{< c8y-admon-req >}}
-This rule is only available if your tenant has a configured SMS provider.
+This rule is available when your tenant has: 
+- configured SMS provider
```
Comment (MWindel) [inline content/cockpit/smart-rules-collection.md]:
```suggestion
- Configured a SMS provider
```

File: content/cockpit/smart-rules-collection.md [hunk 2]
```diff
@@ -25,7 +25,9 @@ In certain rule parameters, various trigger fields can be used as variables, see
...
+- READ permission for the permission type "Option management"
```
Comment (MWindel) [inline content/cockpit/smart-rules-collection.md]:
```suggestion
- The READ permission for the permission type "Option management"
```

File: content/cockpit/smart-rules-collection.md [hunk 3]
```diff
@@ -155,7 +157,9 @@ For details on enabling/disabling a smart rule, see <a href="/cockpit/smart-rule
 If an alarm is created, sends email or SMS.
 
 {{< c8y-admon-req >}}
-This rule is only available if your tenant has a configured SMS provider.
+This rule is available when your tenant has: 
+- configured SMS provider
```
Comment (MWindel) [inline content/cockpit/smart-rules-collection.md]:
```suggestion
- Configured a SMS provider
```

File: content/cockpit/smart-rules-collection.md [hunk 4]
```diff
@@ -155,7 +157,9 @@ For details on enabling/disabling a smart rule, see <a href="/cockpit/smart-rule
...
+- READ permission for the permission type "Option management"
```
Comment (MWindel) [inline content/cockpit/smart-rules-collection.md]:
```suggestion
- The READ permission for the permission type "Option management"
```

**Review by MWindel — APPROVED** [review]

File: content/cockpit/smart-rules-collection.md [hunk 5]
```diff
@@ -25,7 +25,9 @@ In certain rule parameters, various trigger fields can be used as variables, see
 If an alarm is created, an SMS is sent.
 
 {{< c8y-admon-req >}}
-This rule is only available if your tenant has a configured SMS provider.
+This rule is available when your tenant has: 
+- Configured a SMS provider
```
Comment (pmic-cumulo) [inline content/cockpit/smart-rules-collection.md]:
```suggestion
- Your tenant has configured an SMS provider.
```

File: content/cockpit/smart-rules-collection.md [hunk 6]
```diff
@@ -25,7 +25,9 @@ In certain rule parameters, various trigger fields can be used as variables, see
...
+- The READ permission for the permission type "Option management"
```
Comment (pmic-cumulo) [inline content/cockpit/smart-rules-collection.md]:
```suggestion
- Your user has a READ permission for the permission type "Option management".
```

File: content/cockpit/smart-rules-collection.md [hunk 7]
```diff
@@ -155,7 +157,9 @@ For details on enabling/disabling a smart rule, see <a href="/cockpit/smart-rule
 If an alarm is created, sends email or SMS.
 
 {{< c8y-admon-req >}}
-This rule is only available if your tenant has a configured SMS provider.
+This rule is available when your tenant has: 
+- Configured a SMS provider
```
Comment (pmic-cumulo) [inline content/cockpit/smart-rules-collection.md]:
```suggestion
- Your tenant has configured an SMS provider.
```

File: content/cockpit/smart-rules-collection.md [hunk 8]
```diff
@@ -155,7 +157,9 @@ For details on enabling/disabling a smart rule, see <a href="/cockpit/smart-rule
...
+- The READ permission for the permission type "Option management"
```
Comment (pmic-cumulo) [inline content/cockpit/smart-rules-collection.md]:
```suggestion
- Your user has a READ permission for the permission type "Option management".
```

**Comment by pmic-cumulo** [comment]
I need to close this pull request, because it's a "fork". I don't have permission to `c8y-docs`. 
So that's current valid pull request: https://github.com.mcas.ms/SoftwareAG/c8y-docs/pull/2462

**Pull request closed without merging**